packages/sanity/src/core/form/inputs/files/ImageInput/InvalidImageWarning.tsx

```diff
@@ -29,7 +29,12 @@ export function InvalidImageWarning({onClearValue}: Props) {
           <Text size={1}>{t('inputs.files.image.invalid-image-warning.description')}</Text>
```

Comment:
This file uses the wrong namespace (`sanity` instead of `studio`) - could we fix?